annotation-processor/src/main/java/org/hibernate/validator/ap/classchecks/ParametersMethodOverrideCheck.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.ap.classchecks;
+
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.VariableElement;
+import javax.lang.model.util.Elements;
+import javax.lang.model.util.Types;
+
+/**
+ * Checks if the parameters of overridden and overriding methods have correctly placed annotations.
+ * Overriding method cannot have more wider range of constraints on parameters the overridden one.
```

Comment:
-more

on parameters the overridden one -> on parameters than the overridden one